daemons/ipa-kdb/ipa_kdb_mspac.c

```diff
@@ -81,6 +81,18 @@ static struct {
     {NULL, 0}
 };
 
+static struct {
+    char *sid;
+    char *authind;
+} asserted_sids[] = {
+    {"S-1-18-1", "spake"},
+    {"S-1-18-1", "fast"},
```

Comment:
The SPAKE and FAST sids match - is that intentional?  If so, what's S-1-18-2?  (This struct could use a comment either way.)

Reply:
I think we need to find a better fit. S-1-18-1 is "client's identity is asserted by an authentication authority based on proof of possession of client credentials." -- this really fits SPAKE. FAST cannot be used for this.
S-1-18-2 is for S4U2Self case where a service does protocol transition and thus KDC adds S-1-18-2 to say that "the client's identity is asserted by a service."
I think for FAST we should simply set a FAST flag in PAC.

Reply:
"based on proof of possession of client credentials" sounds vague enough that it could be applied to anything Kerberos does :)